hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -764,6 +764,11 @@ public static String getSnapshotPath(OzoneConfiguration conf,
         OM_DB_NAME + snapshotInfo.getCheckpointDirName();
   }
 
+  public static String getSnapshotLocalPropertyYamlPath(OzoneConfiguration conf,
+      SnapshotInfo snapshotInfo) {
+    return getSnapshotPath(conf, snapshotInfo) + ".yaml";
```

Comment:
Shouldn't we be writing the yaml file to a separate metadata dir `om.getMetadataManager().getStore().getSnapshotsParentDir() + "/metadata/<snapshotID>.yaml"` which will be `db.snapshots/checkpointState/metadata/<snapshotID>.yaml` instead of in the snapshotPath `db.snapshots/checkpointState/om.db-<snapshotID>.yaml`?
That way it may be easier to transfer it in the checkpoint tarball during bootstrap.

Reply:
Personally I prefer the latter because it looks cleaner when I do `ls -l` on the `checkpointState` dir, where each snapshot checkpoint would be accompanied by a yaml file right below its listing item. In cases where we have to do manual surgeries it would be easier to locate the yaml (harder to miss).

Reply:
yeah increasing the number of directories can just confuse people